platform/darwin/src/MGLLoggingConfiguration.mm

```diff
@@ -27,8 +67,19 @@ - (void)setHandler:(void (^)(MGLLoggingLevel, NSString *, NSUInteger, NSString *
     }
 }
 
-- (void)logCallingFunction:(const char *)callingFunction functionLine:(NSUInteger)functionLine messageType:(MGLLoggingLevel)type format:(id)messageFormat, ...
-{
+- (void)setLoggingLevel:(MGLLoggingLevel)loggingLevel {
```

Comment:
Is this method needed? Are there cases where the logging level could be outside the bounds?

Reply:
Since `MGLLoggingLevel` is typedef of an integer, there's a potential risk for developers directly set `MGLLoggingLevel` to an actual integer value like `[MGLLoggingConfiguration sharedConfiguration].loggingLevel = -1` and on previously implementation will cause a incorrect filter. However, it seems that there's no problem by filter on platform level. Will delete it